set.mm

```diff
@@ -51077,18 +51076,90 @@ That theorem bundles the theorems ( ` |- E. x ( x = y -> z e. x ) ` with
       ERUGBPEABUBBSTUFRUGAPEABUCASTUD $.
   $}
 
+  ${
+    $d x y z $.
+    $( There exist two sets, one a member of the other.
+
+       This theorem looks similar to ~ el , but its meaning is different.  It
+       only depends on the axioms ~ ax-mp to ~ ax-4 , ~ ax-6 , and ~ ax-pr .
+       This theorem does not exclude that these two sets could actually be one
+       single set containing itself.  That two different sets exist is proved
+       by ~ exexneq .  (Contributed by SN, 23-Dec-2024.) $)
+    exel $p |- E. y E. x x e. y $=
+      ( vz weq wo wel wi wal wex ax-pr ax6ev pm2.07 eximii exim mpi ) ACDZPEZAB
+      FZGAHZRAIZBCCBAJSQAITPQAACKPLMQRANOM $.
+    $( $j usage 'exel' avoids 'ax-5' 'ax-7' 'ax-8' 'ax-9' 'ax-10 'ax-11'
+       'ax-12' 'ax-13''ax-ext' 'ax-sep' 'ax-nul' 'ax-pow'; $)
```

Comment:
Metamath knife panics because of the missing space between `ax-13` and `ax-ext`.
A closing quote is also missing after `ax-10` (here and in below), but that does not seem to be an issue.

Reply:
Clearly Metamath knife should not panic though, but raise an error. I'll open an issue to track this.

Reply:
I've written issue [#174](https://github.com/metamath/metamath-knife/issues/174) of metamath-knife to track this issue, you can follow-up fixes there @benjub.

PS. Sorry I've only had time this weekend to check this in more details.

Reply:
Don't be sorry ! Thanks for fixing this PR and for opening the associated issue.